learning_observer/learning_observer/auth/social_sso.py

```diff
@@ -83,6 +83,30 @@
         'fetch all text from current rosters.',
     default=False
 )
+pmss.register_field(
+    name="default_server",
+    type=pmss.pmsstypes.TYPES.string,
+    description="The Canvas OAuth default server",
```

Comment:
I'm not entirely sure what a _default server_ is in this case. Is this the Authorization URI or make the Token URI? If so, they should follow the same naming conventions as the already used Google ones.

Reply:
Renamed to `token_uri`